lib/msal-browser/docs/testing.md

```diff
@@ -0,0 +1,138 @@
+# Testing
+
+MSAL Browser starting version 2.17.0 has added the `loadTokens()` API, which allows the loading of id tokens and access tokens to the MSAL cache, which can then be read out using `acquireTokenSilent()`. 
+
+**Note: This is an advanced feature that is intended for testing purposes in the browser environment only. Loading tokens to your application's cache may cause your app to break.**
+
+## The loadTokens() API
```

Comment:
I think this heading can go above the two sentences above.

Reply:
Done.